.github/workflows/build.yml

```
@@ -22,46 +23,44 @@ jobs:
       DB_USERNAME: postgres
       DB_PASSWORD: postgres
       DB_HOSTNAME: localhost
+      DOCKER_IMAGE: register-trainee-teacher-data
+      DOCKER_REPO: ${{secrets.DOCKERHUB_USERNAME}}/register-trainee-teacher-data
 
     steps:
     - uses: actions/checkout@v2
 
     - name: Get values for current commit
       run: |
         GIT_SHA_SHORT=$(echo ${{github.sha}} | cut -c 1-7)
-        GIT_REF=${{github.ref}}
-        GIT_BRANCH=${GIT_REF##*/}
+        GIT_REF="${GITHUB_REF}"
+        GIT_BRANCH="${GIT_REF##*/}"
```

`GIT_REF` was an interim variable to get access to `${{github.ref}}`. If we can access it as `$GITHUB_REF` then we can get rid of it:

```suggestion
        GIT_BRANCH="${GITHUB_REF##*/}"
```